frontend/src/pages/artist/_itemId/index.vue

```diff
@@ -123,31 +119,43 @@ import {
 } from '@jellyfin/sdk/lib/generated-client';
 import { getUserLibraryApi } from '@jellyfin/sdk/lib/utils/api/user-library-api';
 import { getItemsApi } from '@jellyfin/sdk/lib/utils/api/items-api';
-import { sanitizeHtml } from '@/utils/html';
-import { getImageInfo, getBlurhash, ImageUrlInfo } from '@/utils/images';
-import { getItemDetailsLink } from '@/utils/items';
+import { getImageInfo, getBlurhash } from '@/utils/images';
 import { msToTicks } from '@/utils/time';
 import { useRemote } from '@/composables';
 
-export default defineComponent({
-  async setup() {
-    const { params } = useRoute();
-    const remote = useRemote();
+const SINGLE_MAX_LENGTH_MS = 600_000;
+const EP_MAX_LENGTH_MS = 1_800_000;
+
+const route = useRoute();
+const remote = useRemote();
+
+const item = ref<BaseItemDto>({});
+const discography = ref<BaseItemDto[]>([]);
+const singles = ref<BaseItemDto[]>([]);
+const eps = ref<BaseItemDto[]>([]);
+const albums = ref<BaseItemDto[]>([]);
+const appearances = ref<BaseItemDto[]>([]);
+const musicVideos = ref<BaseItemDto[]>([]);
+const activeTab = ref(3);
 
-    const albumBreakpoints = {
-      singleMsMaxLength: 600_000,
-      epMsMaxLength: 1_800_000
-    };
-    const itemId = params.itemId;
+const artistBackdrop = computed(() =>
+  getImageInfo(item.value, { preferBackdrop: true })
+);
 
-    const item = (
+watch(
```

Comment:
I don't think this can be written as a computed. This function does a bunch of state assignment which is a side effect and is [not supposed to happen in computed functions](https://vuejs.org/guide/essentials/computed.html#getters-should-be-side-effect-free).